gps_to_enu:
  topic_gps: 'gps/fix'
  topic_publish: 'puma/sensors/gps/enu'
  frame_id: 'map'
  latitude_ref: -33.42196309675523
  longitude_ref: -70.5818018059048
  altitude_ref: 570
  apply_offset_yaw: true

frame_map_odom:
  topic_imu: 'puma/sensors/imu/raw'
  topic_gps_enu: 'puma/sensors/gps/enu'